### ISSUE TITLE
cy.contains problem with extended Latin alphabet characters äöå

### ISSUE THREAD

**Issue by DannyBoyIT** [cypress-io/cypress#8271]
Hello!

I have some problems getting `cy.contains()` to work with some special characters from the Swedish alphabet.

### Current behavior:
My code looks like this:
```js
cy.get('#navigation')
  .contains('Sök')
  .prev()
  .type('test'); //Search for "test".
```

It seems that the characters ä,ö,å cannot be correctly encoded and are presented as bellow:
![image](https://user-images.githubusercontent.com/22036071/90120970-aba98f00-dd5b-11ea-848b-d6556f98b22b.png)

### Desired behavior:
I have some Cypress tests run by Cypress 4.4.0 on another application which do the exact same thing and there the characters are correct and the test is successful as shown by the bellow image:
![image](https://user-images.githubusercontent.com/22036071/90121943-e65ff700-dd5c-11ea-9244-111aad854818.png)

### Test code to reproduce
Include **ö**. **ä** or **å** in a `cy.contains()` to reproduce.

### Versions
- Cypress: 4.12.1
- OS: Windows 10
- Browser: All browsers (Chrome 84, Firefox 75, Edge 84, Electron 80)

**Comment by jennifer-shehane** [comment]
I'm not able to recreate this behavior as described on my Mac in 4.12.1.

<img width="568" alt="Screen Shot 2020-08-14 at 3 46 03 PM" src="https://user-images.githubusercontent.com/1271364/90234196-4af68100-de45-11ea-9994-6004eebc94fd.png">


Is this happening locally on your computer when running or within a Docker container / CI? You may need to ensure you have the correct fonts installed on whatever system you're running your tests within.

**Comment by DannyBoyIT** [comment]
This is happening locally on my Windows 10 computer.
If I run Cypress 4.4.0 on our other project the characters are correct on my computer. When I run our newest project with latest Cypress version the characters are not shown correctly.

**Comment by jennifer-shehane** [comment]
@DannyBoyIT Are you able to narrow down the Cypress version any more on when this broke? There were a lot of releases between 4.4.0 and 4.12.1 - can you try any versions in between to see? That may help narrow down the commit this could have broken within.

**Comment by DannyBoyIT** [comment]
I downgraded to the exact same version as our other project (4.4.0) and I still get the same problem.
I also tested to upgrade our Cypress in the other project to the latest (4.12.1) and it works correctly.

It seems like it is a specific project problem but I can't understand where the problem can be..
If I write a `console.log("ö ä å");` in a normal .js in my solution it works and shows the correct characters.
If I do the same thing in my cypress test file it displays the characters as question marks.
